.github/workflows/test-regression.yaml

```diff
@@ -141,6 +145,10 @@ jobs:
           echo $BUILD_SUCCESS
           echo "build_success=$BUILD_SUCCESS" >> "$GITHUB_OUTPUT"
 
+      - name: Report build success
+        run: |
+          python scripts/post_check_to_patchworks.py -pid $(tail -n 1 patches/${{ inputs.patch_name }})  -desc 'build success' -rid ${{ github.run_id }} -state 'success'
```

Comment:
Add a TODO to link to an issue or github gist. We shouldn't directly link to the run since it's hard to navigate.